### DISCUSSION TITLE
domanda su relè con software Tasmota

### DISCUSSION THREAD

**Discussion by Franco2255** [indomus/forum#2139] [HUB: Home Assistant]
Buongiorno a tutti, come da titolo ho un relè che lavora con un tempo impostato con pulsetime mi chiedevo se per un qualsiasi motivo si spegne il pc sul quale è installato MQTT Broker o la wifi  del router cosa succede il relè si spegne o rimane acceso?
Grazie.

**Comment by snakuzzo** [comment]
Ciao.
Se hai impostato tutto lato device ed in maniera indipendente da publish o subscribe MQTT, non vedo perché debbano esserci impatti sul funzionamento.

**Reply by Franco2255** [reply]
Ok grazie, farò delle prove per sicurezza.
